scripts/ynmp/update.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+import codecs
+import csv
+import json
+import os
+import re
+import sys
+import unicodedata
+import urllib
+
+sys.stdout = codecs.getwriter('utf-8')(sys.stdout)
+
+CSV_URL = 'https://yournextmp.com/media/candidates.csv'
```

Comment:
It'll be faster to use `https://edit.yournextmp.com/media/candidates.csv` directly, so it's not being proxied through the read-only site.